package/autoyast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri Nov 27 16:32:46 CET 2015 - schubi@suse.de
+
+- Added "cobber" to the obsolete profile section.
+  Defined in SUSE Manger but will not be used anymore. (bnc#955878)
```

Comment:
* `cobber` -> `clobber`
* `Manger` -> `Manager`
* `but will` -> `but it will` ?